cmd/common.go

```diff
@@ -34,6 +37,30 @@ func DeleteImages(imageTags []string, imageName string, namespace string, force
 	}
 }
 
+// DeleteResources deletes a list of ConfigMaps or secrets
+func DeleteResources(resources []cfg.Resource, force bool, resourceSelectorFunc cfg.ResourceNamespaceSelector) {
+	if !force {
+		log.Warn("Force mode not enabled, nothing will be deleted")
```

Comment:
We shouldn't treat not-forcing as a problem. The user likely didn't forget, but just performed an (implict because default) dry-run.

A wording like, _"These are the resources we should clean up: ..."_ might be more appropriate.

Reply:
Same here. We'll deal with that in a separate, follow-up PR.